genesis/genesis.go

```diff
@@ -110,7 +118,8 @@ func validateInitialStakedFunds(config *Config) error {
 func validateConfig(networkID uint32, config *Config, stakingCfg *StakingConfig) error {
 	if networkID != config.NetworkID {
 		return fmt.Errorf(
-			"networkID %d specified but genesis config contains networkID %d",
+			"%w both %d and %d",
```

Comment:
would this print `conflicting networkIDs both 1 and 1`? I think that's the wrong error here

Reply:
This is only returned if `networkID != config.NetworkID`